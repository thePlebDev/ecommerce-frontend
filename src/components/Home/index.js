import React from 'react'



const Home = ()=>{


  return(
    <div>
      <h2 className="home">Home</h2>
    </div>
  )
}

export default Home
