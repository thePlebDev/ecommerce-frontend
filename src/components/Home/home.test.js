import React from 'react'
import { shallow } from 'enzyme'

import Home from './index.js'

it('should render ',()=>{
  const component = shallow(<Home />)

  expect(component.getElements()).toMatchSnapshot()

});
