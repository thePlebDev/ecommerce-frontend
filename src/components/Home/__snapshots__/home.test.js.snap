// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render  1`] = `
Array [
  <div>
    <h2
      className="home"
    >
      Home
    </h2>
  </div>,
]
`;
